Dockerfile

```diff
@@ -45,6 +45,9 @@ COPY guacamole-docker/bin/ /opt/guacamole/bin/
 COPY . "$BUILD_DIR"
 
 # Run the build itself
+RUN ["chmod", "777", "/opt/guacamole/bin/build-guacamole.sh"]
+RUN ["chmod", "777", "/opt/guacamole/bin/start.sh"]
```

Comment:
These changes seem out-of-scope for this particular pull request.  If the pull request is focused on modifying how the GUAC_DATE and GUAC_TIME tokens behave, then it should be kept to that, without these extra changes.

Reply:
These changes would also add far more permissions to the scripts than they should be given. The build and startup scripts definitely do not need to be world-writable.